neuro_config_client/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Platform config client."""
-from pkg_resources import get_distribution
```

Comment:
why do we need this change?

Reply:
`pkg_resources` is [deprecated](https://setuptools.pypa.io/en/latest/pkg_resources.html) and the corresponding warnings are breaking our tests here and in neuro-cli.
In CLI I was forced to [ignore](https://github.com/neuro-inc/neuro-cli/pull/2908/commits/74c3fb37a894cf515221e026e3abc04d2d4079d3) those warnings, but here decided to resolve this problem.